exchanges/bittrex/bittrex_wrapper.go

```diff
@@ -954,86 +953,39 @@ func (b *Bittrex) FormatExchangeKlineInterval(in kline.Interval) string {
 // This implementation rounds returns candles up to the next interval or to the end
 // time (whichever comes first)
 func (b *Bittrex) GetHistoricCandles(ctx context.Context, pair currency.Pair, a asset.Item, interval kline.Interval, start, end time.Time) (*kline.Item, error) {
-	req, err := b.GetKlineRequest(pair, a, interval, start, end)
+	req, err := b.GetKlineRequest(pair, a, interval, start, end, false)
```

Comment:
Remember how I just got you to change this function? Well egg on my face. Despite my testing, looks as though bittrex likes to cut off data at intervals. eg getting 1 year of data from this point last year until now goes until 31/12/2022.
I personally don't see the need to make this function return two requests in the event of date rollover, that's what `GetHistoricCandlesExtended` is for. But I understand if you wish to revert my suggestion.

Reply:
Yeah I had to revert to keep it in line with other implementations.